### ISSUE TITLE
`jsdoc/check-line-alignment` incorrectly triggered if `@returns` description starts with `-1`

### ISSUE THREAD

**Issue by willeastcott** [gajus/eslint-plugin-jsdoc#1091]
It seems that the `jsdoc/check-line-alignment` rule is triggered if an `@returns` description begins with `-1` (number).

## Expected behavior
AFAIK, the dash is not recommended to delimit the @returns type from the description. So I would expect this to not trigger the error.

## Actual behavior

Error is output:

```
C:\dev\engineclean\src\scene\graph-node.js
  915:1  error  Expected JSDoc block lines to not be aligned  jsdoc/check-line-alignment

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

## ESLint Config

Include this:

```
"jsdoc/check-alignment": "error"
```

This appears in this full config:

https://github.com/playcanvas/playcanvas-eslint-config/blob/9dac541d64a73876ed27d5298eed7197981e6958/index.js#L400

## ESLint sample

This code triggers the error:

```js
    /**
     * Returns cached value of negative scale of the world transform.
     *
     * @returns {number} -1 if world transform has negative scale, 1 otherwise.
     * @ignore
     */
    get worldScaleSign() {

        if (this._worldScaleSign === 0) {
            this._worldScaleSign = this.getWorldTransform().scaleSign;
        }

        return this._worldScaleSign;
    }
```

Error output is:

![image](https://github.com/gajus/eslint-plugin-jsdoc/assets/697563/14bcfb43-0d8d-4fe8-80ed-bc8788275450)

Doing an autofix will update this:

```js
     * @returns {number} -1 if world transform has negative scale, 1 otherwise.
```

To this:

```js
     * @returns {number} - 1 if world transform has negative scale, 1 otherwise.
```

This completely changes the meaning of the docs, of course.

## Environment

- Node version: v18.13.0
- ESLint version: v8.41.0
- `eslint-plugin-jsdoc` version: v44.2.4
